persistence/sql/persister_session.go

```diff
@@ -171,7 +163,7 @@ func (p *Persister) ListSessionsByIdentity(
 		}
 		t = int64(total)
 
-		q.Order("authenticated_at DESC")
+		q.Order("created_at DESC")
```

Comment:
Is there no other way to do this? IMO, `authenticated_at` is a more useful sorting order. Plus this is breaking.

Reply:
created_at == authenticated_at with maybe ~300ms clock skew